File: app/buildz/tmp/kapt3/stubs/debug/com/bits/bee/bpmc/utils/BPMConstants.java
package com.bits.bee.bpmc.utils;

import java.lang.System;

/**
 * Created by aldi on 01/03/22.
 */
@kotlin.Metadata(mv = {1, 7, 1}, k = 1, d1 = {"\u0000\f\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\b\u0002\bf\u0018\u0000 \u00022\u00020\u0001:\u0001\u0002\u00a8\u0006\u0003"}, d2 = {"Lcom/bits/bee/bpmc/utils/BPMConstants;", "", "Companion", "app_debug"})
public abstract interface BPMConstants {
    @org.jetbrains.annotations.NotNull()
    public static final com.bits.bee.bpmc.utils.BPMConstants.Companion Companion = null;
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_POSSES = "POSSES";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_SALE = "SALE";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CASH = "CASH";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CADJ = "CADJ";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CSTR = "CADJ";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_TUNAI = "TUNAI";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CHG = "CHG";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_DEBIT = "DC";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_KREDIT = "CC";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String CADJ = "CADJ";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String SALE = "SALE";
    public static final int BPM_LIMIT_PAGINATION = 10;
    public static final int BPM_MAX_PAGINATION = 30;
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CASH_POSSES = "POSSES";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String CREDIT_CARD_CODE = "CC";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String DEBIT_CARD_CODE = "DC";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String BPM_DEFAULT_TYPE_CASH_GOPAY = "GOPAY";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String SCREEN_POTRAIT = "Portrait";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String SCREEN_LANDSCAPE = "Landscape";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String DEFAULT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSSSSS";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String NEW_DATE_FORMAT = "dd/MM/yyyy";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String MODE_FOOD_BEVERAGES = "MODE_FOOD_AND_BEVERAGES";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String MODE_RETAIL = "MODE_RETAIL";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String NOTASETTING_LOGOPATH = "NOTASETTING_LOGOPATH";
    @org.jetbrains.annotations.NotNull()
    public static final java.lang.String INIT_DEFAULT_LOGO = "bpm_logo1";
    
    @kotlin.Metadata(mv = {1, 7, 1}, k = 1, d1 = {"\u0000$\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0002\b\u0002\n\u0002\u0010\u000e\n\u0002\b\u000b\n\u0002\u0010\b\n\u0002\b\u0007\n\u0002\u0018\u0002\n\u0002\b\n\b\u0086\u0003\u0018\u00002\u00020\u0001B\u0007\b\u0002\u00a2\u0006\u0002\u0010\u0002R\u000e\u0010\u0003\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0005\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0006\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0007\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\b\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\t\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\n\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u000b\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\f\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\r\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u000e\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u000f\u001a\u00020\u0010X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0011\u001a\u00020\u0010X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0012\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0013\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0014\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0015\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u0016\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u0011\u0010\u0017\u001a\u00020\u0018\u00a2\u0006\b\n\u0000\u001a\u0004\b\u0019\u0010\u001aR\u000e\u0010\u001b\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u001c\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u001d\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u001e\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010\u001f\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010 \u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000R\u000e\u0010!\u001a\u00020\u0004X\u0086T\u00a2\u0006\u0002\n\u0000\u00a8\u0006\""}, d2 = {"Lcom/bits/bee/bpmc/utils/BPMConstants$Companion;", "", "()V", "BPM_DEFAULT_TYPE_CADJ", "", "BPM_DEFAULT_TYPE_CASH", "BPM_DEFAULT_TYPE_CASH_GOPAY", "BPM_DEFAULT_TYPE_CASH_POSSES", "BPM_DEFAULT_TYPE_CHG", "BPM_DEFAULT_TYPE_CSTR", "BPM_DEFAULT_TYPE_DEBIT", "BPM_DEFAULT_TYPE_KREDIT", "BPM_DEFAULT_TYPE_POSSES", "BPM_DEFAULT_TYPE_SALE", "BPM_DEFAULT_TYPE_TUNAI", "BPM_LIMIT_PAGINATION", "", "BPM_MAX_PAGINATION", "CADJ", "CREDIT_CARD_CODE", "DEBIT_CARD_CODE", "DEFAULT_DATE_FORMAT", "INIT_DEFAULT_LOGO", "MC_FOUR", "Ljava/math/MathContext;", "getMC_FOUR", "()Ljava/math/MathContext;", "MODE_FOOD_BEVERAGES", "MODE_RETAIL", "NEW_DATE_FORMAT", "NOTASETTING_LOGOPATH", "SALE", "SCREEN_LANDSCAPE", "SCREEN_POTRAIT", "app_debug"})
    public static final class Companion {
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_POSSES = "POSSES";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_SALE = "SALE";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CASH = "CASH";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CADJ = "CADJ";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CSTR = "CADJ";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_TUNAI = "TUNAI";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CHG = "CHG";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_DEBIT = "DC";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_KREDIT = "CC";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String CADJ = "CADJ";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String SALE = "SALE";
        public static final int BPM_LIMIT_PAGINATION = 10;
        public static final int BPM_MAX_PAGINATION = 30;
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CASH_POSSES = "POSSES";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String CREDIT_CARD_CODE = "CC";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String DEBIT_CARD_CODE = "DC";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String BPM_DEFAULT_TYPE_CASH_GOPAY = "GOPAY";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String SCREEN_POTRAIT = "Portrait";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String SCREEN_LANDSCAPE = "Landscape";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String DEFAULT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss.SSSSSS";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String NEW_DATE_FORMAT = "dd/MM/yyyy";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String MODE_FOOD_BEVERAGES = "MODE_FOOD_AND_BEVERAGES";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String MODE_RETAIL = "MODE_RETAIL";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String NOTASETTING_LOGOPATH = "NOTASETTING_LOGOPATH";
        @org.jetbrains.annotations.NotNull()
        public static final java.lang.String INIT_DEFAULT_LOGO = "bpm_logo1";
        @org.jetbrains.annotations.NotNull()
        private static final java.math.MathContext MC_FOUR = null;
        
        private Companion() {
            super();
        }
        
        @org.jetbrains.annotations.NotNull()
        public final java.math.MathContext getMC_FOUR() {
            return null;
        }
    }
}

File: app/src/main/java/com/bits/bee/bpmc/utils/BPMConstants.kt
package com.bits.bee.bpmc.utils

import android.os.Build
import android.os.Environment
import java.math.MathContext
import java.math.RoundingMode

/**
 * Created by aldi on 01/03/22.
 */
interface BPMConstants {
    companion object {
        const val AWS_KEY = "DZA3URDW4JHRENEH2D5V"
        const val AWS_SECRET = "c2i6w3QR5/nHq2yIkuHDssBKBOtgci/eNsAi+5jUzdI"
        const val BPM_DEFAULT_TYPE_POSSES = "POSSES"
        const val BPM_DEFAULT_TYPE_SALE = "SALE"
        const val BPM_DEFAULT_TYPE_CASH = "CASH"
        const val BPM_DEFAULT_TYPE_CADJ = "CADJ"
        const val BPM_DEFAULT_TYPE_CSTR = "CSTR"
        const val BPM_DEFAULT_TYPE_TUNAI = "TUNAI"
        const val BPM_DEFAULT_TYPE_CHG = "CHG"
        const val BPM_DEFAULT_TYPE_DEBIT = "DC"
        const val BPM_DEFAULT_TYPE_KREDIT = "CC"
        const val CADJ = "CADJ"
        const val SALE = "SALE"
        const val BPM_LIMIT_PAGINATION = 10
        const val BPM_MAX_PAGINATION = 30
        const val BPM_DEFAULT_TYPE_CASH_POSSES = "POSSES"
        const val CREDIT_CARD_CODE = "CC"
        const val DEBIT_CARD_CODE = "DC"
        const val BPM_DEFAULT_TYPE_CASH_GOPAY = "GOPAY"
        const val SCREEN_POTRAIT = "Portrait"
        const val SCREEN_LANDSCAPE = "Landscape"
        const val DATE_FORMAT_RESPONSE = "yyyy-MM-dd HH:mm:ss.SSSSSS"
        const val DEFAULT_DATE_FORMAT = "yyyy-MM-dd HH:mm:ss"
        const val NEW_DATE_FORMAT = "dd/MM/yyyy"
        const val MODE_FOOD_BEVERAGES = "MODE_FOOD_AND_BEVERAGES"
        const val MODE_RETAIL = "MODE_RETAIL"
        const val NOTASETTING_LOGOPATH = "NOTASETTING_LOGOPATH"
        const val INIT_DEFAULT_LOGO = "bpm_logo1"
        const val API_MY = "http://my.beecloud.id"
        const val DEFAULT_DB_OFFSET: Long = 0
        const val DEFAULT_DB_LIMIT: Long = 17
        const val DEFAULT_DB_UPLOAD: Long = 40
        const val BPM_DEFAULT_LICENSE_TYPE = "MOBILE"
        const val REGEX_INPUT = "[().,*#!?/|:;'<>@$%&^=+-]"
        const val DIVIDE_DIALOG_SIZE = 2.0
        const val REG_ROUND = "ROUND"
        const val REG_SALE_TAXED = "SALE_TAXED"
        const val REG_SALE_TAXINC = "SALE_TAXINC"
        const val REG_POSSES_ACTUAL_ENDCASH = "POSSES_ACTUAL_ENDCASH"
        const val REG_POS_CARDNO_REQUIRED = "POS_CARDNO_REQUIRED"
        const val REG_POS_TRACKNO_REQUIRED = "POS_TRACKNO_REQUIRED"
        const val REG_ROUND_DIGITS = "ROUND_DIGITS"
        const val REG_ROUND_DIGITS_ENB = "ROUND_DIGITS_ENB"
        const val REG_ROUND_DOWN_BLW = "ROUND_DOWN_BLW"
        const val REG_PRCLVL_DEFAULT = "PRCLVL_DEFAULT"
        const val REG_CHANNEL_PRICE = "CHANNEL_PRICE"
        const val BPM_PARENTPATH = "/BPM"
        const val BPM_PRODUKPATH = "$BPM_PARENTPATH/PRODUK"

        fun getDatapath(): String{
            return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES)
                    .toString()
            } else {
                StringBuilder(Environment.getExternalStorageDirectory().absolutePath.toString()).toString()
            }
        }

        const val BPM_TYPE_ROUNDING = "ROUNDING"

        // PRIVILLEGE
        const val PRVSALE = "815005"
        const val PRVSO = "815002"
        const val ACS_ENB = "ENB"
        const val ACS_NEW = "NEW"
        const val ACS_OPN = "OPN"
        const val ACS_UPD = "UPD"
        const val ACS_DEL = "DEL"
        const val ACS_PRN = "PRN"
        const val ACS_DISC = "DISC"
        const val ACS_DISC_MASTER = "DISC_MASTER"
        const val ACS_TAXINC = "TAXINC"
        const val ACS_PRICE_EDIT = "PRICE_EDIT"
        const val BPM_PRIVILEGE_PRICE = "PRICE_EDIT"
        const val BPM_PRIVILEGE_DISC = "DISC"
        const val BPM_PRIVILEGE_DISC_MASTER = "DISC_MASTER"
        const val BPM_PRIVILEGE_OBJ = "845005"

        const val CHAR_ENTER = "\n"
        const val CHAR_SPACE = " "
        const val CHAR_DOUBLE_SPACE = "  "
        const val CHAR_TRIPLE_SPACE = "   "
        const val CHAR_LONG_SPACE = "     "
        const val CHAR_DOUBLE_ENTER = "\n\n"

        const val BPM_TAB_SMALL = 5
        const val BPM_TAB_NORMAL = 3
        const val BPM_TAB_LARGE = 1
        const val BPM_TAB_HEADER_SMALL = 9
        const val BPM_TAB_HEADER_NORMAL_USB = 6
        const val BPM_TAB_HEADER_NORMAL = 7
        const val BPM_TAB_HEADER_LARGE = 5
        const val BPM_TAB_HEADER_LARGE_USB = 3

        const val BPM_FONT_REGULAR = "REGULAR"
        const val BPM_ALIGN_LEFT = "LEFT"
        const val BPM_ALIGN_CENTER = "CENTER"
        const val BPM_ALIGN_RIGHT = "RIGHT"

        const val BPM_DEFAULT_THREAD_TIMEOUT: Long = 100000
        const val BPM_INIT_DEFAULT_LOGO = "logo_bpmc_01"

        const val BPM_PRINTER_BLUETOOH = 0
        const val BPM_PRINTER_WIFI = 1
        const val BPM_PRINTER_USB = 2

        val MC_FOUR: MathContext = MathContext(19, RoundingMode.HALF_UP)

        const val NOTIFICATION_ID = 833
    }
}